app/views/posts/index.html.erb

```diff
@@ -29,7 +29,7 @@
       <h3>Posts tagged: <%= tag.name %></h3>
       <% posts.each do |post| %>
         <%# Template Dependency: post/post_subject %>
-        <% cache(post, namespace: 'post_threads') do %>
+        <% cache_unless(post.seeing_activity?, post, namespace: 'post_threads') do %>
```

Comment:
Should we really be doing cache logic in an .erb? This seems like it really should be managed by a controller instead...

Reply:
Is the objection to caching in the views generally or to there being a conditional there?